src/components/Button/package.json

```diff
@@ -28,7 +28,7 @@
     "@material-ui/core": "^4.11.0"
   },
   "devDependencies": {
-    "@material-ui/icons": "^4.11.2"
+    "@gemeente-denhaag/icons": "*"
```

Comment:
I don't think we want unlimited version ranges

Reply:
It is a devDependency because it is only used in storybook, where you always want the latest value so it should be fine.